src/main/kotlin/no/nav/familie/oppdrag/tilbakekreving/ØkonomiConsumer.kt

```diff
@@ -0,0 +1,59 @@
+package no.nav.familie.oppdrag.tilbakekreving
+
+import no.nav.familie.oppdrag.config.IntegrasjonException
+import no.nav.okonomi.tilbakekrevingservice.KravgrunnlagHentDetaljRequest
+import no.nav.okonomi.tilbakekrevingservice.KravgrunnlagHentDetaljResponse
+import no.nav.okonomi.tilbakekrevingservice.TilbakekrevingPortType
+import no.nav.okonomi.tilbakekrevingservice.TilbakekrevingsvedtakRequest
+import no.nav.okonomi.tilbakekrevingservice.TilbakekrevingsvedtakResponse
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import org.springframework.stereotype.Service
+import java.math.BigInteger
+import java.util.UUID
+import javax.xml.ws.soap.SOAPFaultException
+
+@Service
+class ØkonomiConsumer(private val økonomiService: TilbakekrevingPortType) {
```

Comment:
`TilbakekrevingService` ?

Reply:
ØkonomiClient
TilbakekrevingClient
???

Reply:
Vi brukte det samme navn på familie-tilbake. Siden familie-tilbake kun skal bruke det, tror det er nyttig å bruke samme navn

Reply:
Jeg tenker vi burde forholde oss til en app sin navnestandard :)

Reply:
TilbakekrevingClient høres litt forvirring ut fordi vi ikke kaller familie-tilbake derfra. Men det kan endres til ØkonomiClient og ØkonomiService siden vi kommuniserer med Økonomi

Reply:
Dere kaller på en endepunkt for tilbakekreving fra økonomi, `TilbakekrevingPortType`, så tenker det er helt greit å kalle den for noe med Tilbakekreving